src/runtime/composables/cwa-resource.ts

```diff
@@ -7,13 +7,14 @@ export type IriProp = {
 }
 
 interface cwaResourceUtilsOps {
```

Comment:
Let's start to put manager options into a `manager` property
```
interface CwaResourceManagerOptions {
  displayName?: string
}
...
manager: {
  disabled?: boolean
  options?: CwaResourceManagerOptions
}
```

You can use CwaResourceManagerOptions for passing to `useCwaResourceManageable` composable too

Reply:
This is the comment re the manager ops and CwaResourceManagerOptions

Reply:
Sorry typo on the name probably more like ManageableComponentOptions